pom.xml

```diff
@@ -37,84 +38,89 @@
         by supplying input streams and dictionaries.
     </description>
 
+    <properties>
+        <sling.java.version>8</sling.java.version>
+    </properties>
     <scm>
         <connection>scm:git:https://gitbox.apache.org/repos/asf/sling-org-apache-sling-installer-core.git</connection>
         <developerConnection>scm:git:https://gitbox.apache.org/repos/asf/sling-org-apache-sling-installer-core.git</developerConnection>
         <url>https://gitbox.apache.org/repos/asf?p=sling-org-apache-sling-installer-core.git</url>
-      <tag>HEAD</tag>
-  </scm>
+        <tag>HEAD</tag>
+    </scm>
 
-	<build>
-		<plugins>
-			<plugin>
-				<groupId>org.apache.felix</groupId>
-				<artifactId>maven-bundle-plugin</artifactId>
-				<extensions>true</extensions>
-				<configuration>
-					<instructions>
-						<Bundle-Activator>
-							org.apache.sling.installer.core.impl.Activator
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-Activator>
+                            org.apache.sling.installer.core.impl.Activator
                         </Bundle-Activator>
                         <Conditional-Package>
                             org.apache.felix.configurator.impl.json,
-                            org.apache.felix.configurator.impl.model                            
+                            org.apache.felix.configurator.impl.model,
+                            org.apache.sling.feature,
+                            org.apache.sling.feature.*
                         </Conditional-Package>
                         <Embed-Dependency>
-                            geronimo-json_1.0_spec,johnzon-core,org.apache.felix.converter,
```

Comment:
@cziegeler Was there a reason JSON-P was embedded? I ran into the issue mentioned at https://issues.apache.org/jira/browse/GERONIMO-6560 with these dependencies embedded! Was it only there to keep the dependencies to a minimum? Just relying on the bundle https://github.com/apache/sling-org-apache-sling-commons-johnzon should be fine IMHO.

Reply:
yes, the reason was to keep the dependencies to a minimum.

Reply:
Either we need to rely on Aries SPI Fly or embed Sling Commons JSON, otherwise we run into classloader issues.

Reply:
and if we just make it a regular dependency?

Reply:
I think embedding is fine (https://github.com/apache/sling-org-apache-sling-installer-core/pull/5/commits/9cf63dfb448729d91272b14c5979c632ce1f046b). The total size of the bundle is now 400kB. Currently in Sling Johnzon is loaded with StartLevel 5 (https://github.com/apache/sling-org-apache-sling-starter/blob/9898f313147e97e15721b71198c5196d4b4a1262/src/main/provisioning/sling.txt#L42)